src/Components/Web.Extensions/src/HeadManagement/HeadManagementJSRuntimeExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
```

Comment:
For all the files added in this PR, could you add the standard header at the top? Thanks!

```
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```